ocpp_codec/serializer.py

```diff
@@ -34,20 +34,25 @@ def _required_fields(dataclass_class) -> typing.List[dataclasses.Field]:
 
 
 def _run_validator(field: dataclasses.Field, data: typing.Any, *, parsing: bool) -> typing.Any:
-    # Fetch the validator to run
-    validator = field.metadata.get('validators', validators.noop)
-    if isinstance(validator, validators.BaseEncoder):
-        # Pick the right method based on whether we're parsing a message, or serializing a field
-        validator = validator.from_json if parsing else validator.to_json
+    # Fetch the validators to run
+    validator_list = field.metadata.get('validators', [validators.noop])
+    # Defining a single validator is supported, turn it to a list for easier processing
+    if not isinstance(validator_list, list):
+        validator_list = [validator_list]
```

Comment:
Having a metadata called `validators` in which we allow a non-list value sounds fishy to me.
I think we should have lists everywhere for consistency.

Reply:
Also, we could let users use any enumeration type like tuples which would me more appropriate in this case, what do you think?

Reply:
Yeah, I agree on the list-everywhere thing.